### DISCUSSION TITLE
My partner portal URL returns an error

### DISCUSSION THREAD

**Discussion by gentoog** [community/community#38823] [GitHub Education] [answered]
I received a partner portal URL through a scholarship but when I submit it, it gives an error: Something went wrong with the specified school.

The administrator cannot troubleshoot. How can I get this fixed?

**Comment by grahamzemel** [comment, marked as answer]
If you received a partner URL, see if you can censor any sensitive data and comment it here. It's likely that a request is being sent with malformed data that causes a page error (or, if it's possible, the school may not be entered into the scholarship's database).   

With a bit more information this is likely a quick fix, and if you're wary of sharing data on public forms, here's [my contact](https://grahamzemel.com/contact).

**Reply by gentoog** [reply]
Thanks! I will reply to contact

**Reply by samus-aran** [reply]
Although we appreciate your participation here @grahamzemel we highly discourage our members to directly contact other GitHub users with potentially sensitive information concerning their issue.

If students are having issues with their application we will do what we can to assist within the public discussions. If there is a need for transfer of sensitive data then we will direct them to the appropriate support team to continue the conversation. 👍

**Comment by samus-aran** [comment]
Hi @gentoog 👋 

How did you get on or are you still having issues?

**Comment by magarsaakaamil9-dot** [comment]
History is disqualified results please help me
